ts/src/oxfun.ts

```diff
@@ -0,0 +1,2843 @@
+        /**
+         * @method
+         * @name oxfun#createOrder
+         * @description create a trade order
+         * @see https://docs.ox.fun/?json#post-v3-orders-place
+         * @param {string} symbol unified symbol of the market to create an order in
+         * @param {string} type 'market', 'limit', 'STOP_LIMIT' or 'STOP_MARKET'
+         * @param {string} side 'buy' or 'sell'
+         * @param {float} amount how much of currency you want to trade in units of base currency
+         * @param {float} [price] the price at which the order is to be fullfilled, in units of the quote currency, ignored in market orders
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @param {int} [params.clientOrderId] a unique id for the order
+         * @param {int} [params.timestamp] in milliseconds. If an order reaches the matching engine and the current timestamp exceeds timestamp + recvWindow, then the order will be rejected.
+         * @param {int} [params.recvWindow] in milliseconds. If an order reaches the matching engine and the current timestamp exceeds timestamp + recvWindow, then the order will be rejected. If timestamp is provided without recvWindow, then a default recvWindow of 1000ms is used.
+         * @param {string} [params.responseType] FULL or ACK
+         * @param {float} [params.cost] the quote quantity that can be used as an alternative for the amount for market buy orders
+         * @param {float} [params.triggerPrice] The price at which a trigger order is triggered at
+         * @param {float} [params.limitPrice] Limit price for the STOP_LIMIT order
+         * @param {bool} [params.postOnly] if true, the order will only be posted if it will be a maker order
+         * @param {string} [params.timeInForce] GTC (default), IOC, FOK, PO, MAKER_ONLY or MAKER_ONLY_REPRICE (reprices order to the best maker only price if the specified price were to lead to a taker trade)
+         * @param {string} [params.selfTradePreventionMode] NONE, EXPIRE_MAKER, EXPIRE_TAKER or EXPIRE_BOTH for more info check here {@link https://docs.ox.fun/?json#self-trade-prevention-modes}
+         * @param {string} [params.displayQuantity] for an iceberg order, pass both quantity and displayQuantity fields in the order request
+         * @returns {object} an [order structure]{@link https://docs.ccxt.com/#/?id=order-structure}
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request: Dict = {
+            'responseType': this.safeString (params, 'responseType', 'FULL'),
+            'timestamp': this.safeInteger (params, 'timestamp', this.milliseconds ()),
+        };
+        params = this.omit (params, [ 'responseType', 'timestamp' ]);
+        const recvWindow = this.safeInteger (params, 'recvWindow');
```

Comment:
@rayBastard @yzh-pelle shouldn't timestamp/recvwindow be handled inside sign()?

Reply:
It is another `timestamp` for creating an order. Also `recwWindow` is used only for orders, other `private` endpoints don't accept it

Reply:
ok ok